src/core/agent/services/identifierService.test.ts

```diff
@@ -108,53 +166,35 @@ describe("Identifier service of agent", () => {
         displayName: "Identifier 2",
         signifyName: "uuid-here",
         colors,
-
         createdAtUTC: nowISO,
         theme: 0,
         isPending: false,
       },
     ]);
   });
-
   test("can get all identifiers without error if there are none", async () => {
-    identifierService.getAllIdentifierMetadata = jest
+    identifierStorage.getAllIdentifierMetadata = jest
       .fn()
       .mockResolvedValue([]);
     expect(await identifierService.getIdentifiers()).toStrictEqual([]);
   });
 
-  test("search for non existant keri aid (in db)", async () => {
```

Comment:
Why was this test removed?

Reply:
We have another test in storage. Don't need to check again